scripts/python/run_zfs_remote_replication.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+import sys
+from integralstor_utils import zfs
+from integralstor import remote_replication
+from datetime import datetime
+
+
+def run_zfs_remote_replication(remote_replication_id):
+    try:
+        rr, err = remote_replication.get_remote_replications(
+            remote_replication_id)
+        if err:
+            raise Exception('Could not fetch replication details: %s' % err)
+        replication = rr[0]
+        mode = replication['mode']
+
+        if mode == 'zfs':
+            zfs_entries = replication['zfs'][0]
+            source_dataset = zfs_entries['source_dataset']
+            target_ip = zfs_entries['target_ip']
+            target_pool = zfs_entries['target_pool']
+            target_user_name = zfs_entries['target_user_name']
+            description = replication['description']
+            now = datetime.now()
+            now_str = now.strftime('%Y-%m-%d-%H-%M')
```

Comment:
Can you change this to use the datetime_utils and pull the local or utc  timestamp, whichever makes more sense?